templates/angular/igx-ts/grid/grid-custom/index.ts

```diff
@@ -159,5 +171,11 @@ class IgxCustomGridTemplate extends IgniteUIForAngularTemplate {
 		};
 		return super.generateFiles(projectPath, name, { extraConfig });
 	}
+
+	private formatGridFeatures(feature: string, columnFeatures: any[], columnBoolFeatures: any[]) {
+		const text = `[${feature.toLowerCase().replace("ing", "able")}]="true"`;
+		columnFeatures.push(text);
+		columnBoolFeatures.push(text);
```

Comment:
This will push all features equally to all columns, including the boolean template one and that one was separated for a reason - it can't be editable (the feature)